Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/TabbedPage.cs

```diff
@@ -150,52 +151,71 @@ public static int GetMaxItemCount(this IPlatformElementConfiguration<Android, Fo
 			return GetMaxItemCount(config.Element);
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		[Obsolete("This property is obsolete as of version 3.5. Please use TabbedPage.UnselectedTabColor instead.")]
 		public static readonly BindableProperty BarItemColorProperty =
 			BindableProperty.Create("BarItemColor", typeof(Color),
 			typeof(TabbedPage), Color.Default);
 
-
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		[Obsolete("This property is obsolete as of version 3.5. Please use TabbedPage.UnselectedTabColor instead.")]
 		public static Color GetBarItemColor(BindableObject element)
 		{
-			return (Color)element.GetValue(BarItemColorProperty);
+			return (Color)element.GetValue(FormsElement.UnselectedTabColorProperty);
```

Comment:
I was thinking to keep the old way, but add a `propertyChanged: OnChange` bit to the binding property above. This way we keep them in sync.